src/distilabel/models/llms/mlx.py

```diff
@@ -42,7 +44,7 @@ class MlxLLM(LLM, MagpieChatTemplateMixin):
     Attributes:
         path_or_hf_repo: the path to the model or the Hugging Face Hub repo id.
         tokenizer_config: the tokenizer configuration.
-        model_config: the model configuration.
+        mlx_model_config: the MLX model configuration.
```

Comment:
We couldn't use `model_config` as it's a reserved attribute of `pydantic.BaseModel` class.